docs/flow/modin/core/dataframe/base/index.rst

```diff
@@ -1,3 +1,17 @@
+Purpose
+=======
+``BaseDataframe`` serves the purpose of describing and defining the internal dataframe algebra.
+
+It is the core construction element which serves as a middle layer between high-level pandas-imitating API
```

Comment:
I do not quite get this paragraph. You are saying here that base dataframe is a middle layer between API layer and QC layer whereas QC layer is actually a middle layer between API layer and base/core dataframe layer.

Reply:
Huh, now you wrote it I agree... I might've been fooled by some other diagram I have interpreted incorrectly.